Fandom Creator/JavaScript.md

```diff
@@ -6,6 +6,8 @@ These coding conventions are for JavaScript inside the Fandom Creator Repo.
 | --- | ----------- |
 | [Binding this](#binding-this) | MUST |
 | [Static Data Within a Class](#static-data-within-a-class) | SHOULD |
+| [Instance fields in a class](#instance-fields-in-a-class) | MUST |
```

Comment:
Hmm I don't find this table format useful. I think it would be better if we include the MAY, SHOULD, or MUST within the descriptive text below. Even if we want to leave the the table as a shortcut, we should still be using the MAY/SHOULD/MUST keywords (and in all caps) in the described guidelines.

Reply:
d805e13bd15d42d4865c447fa4c536760dccb082